pages/common/firefox.md

```diff
@@ -27,6 +27,14 @@
 
 `firefox --headless --screenshot {{path/to/output_file.png}} {{https://example.com/}}`
 
-- Use a specific profile directory to allow multiple separate instances of Firefox to run at once:
+- Use a specific profile directory to allow multiple separate instances of Firefox to run at once (not available for Windows):
```

Comment:
Well now this is interesting. I feel like we shouldn't have an example that is not available to all platforms in the `common` directory.
If thats the case we have two options:
1. Keep this `common` page, while creating a new page under `windows` but without this example. 
2. Delete this page, recreate it under `linux` (and maybe macOS too?) while also creating a new page under `windows` without this example. 

What's the way to go here, guys?